src/main/webapp/js/utils/form/form-utils.jsx

```diff
@@ -74,6 +76,12 @@ const renderFormField = (props) => {
       FieldType = DateField;
       attr = { dateFormat: format };
       break;
+    case TIME:
+      FieldType = TimeField;
+      attr = {
+        timeFormat: format, showTimeSelect: true, showTimeSelectOnly: true, withPortal: true,
```

Comment:
I think you should remove the showTimeSelect and showTimeSelectOnly, because it should be always true for the time field